### ISSUE TITLE
Update node engine in package.json

### ISSUE THREAD

**Issue by chicoxyzzy** [compat-table/compat-table#994]
We have [node 0.11.12](https://github.com/kangax/compat-table/blob/gh-pages/package.json#L6) as a lower node version and that's weird. Should we change it to 4.7.0 or maybe even 0.10.48?

**Comment by ljharb** [comment]
Indeed, it should be as low as is required. However, I'm not sure if `npm run build` still works on node 0.10 - can someone verify the lowest node version it works on, and we'll adjust the engines field to match?

The original [commit](https://github.com/kangax/compat-table/commit/cd2e9be2915f74a4f81390461fe95482950c9ecf) that added this, however, indicates why it requires 0.11.12 or later - so I think that the only thing we'd need to do is raise it higher, if in fact it doesn't still work on 0.11.12.

**Comment by chicoxyzzy** [comment]
`npm run build` and `npm run build:compilers` both work on v0.10.48. Would be great if @webbedspace will chime in.

**Comment by zloirock** [comment]
I don't think that it make any sense. We could just remove it.

**Comment by ljharb** [comment]
"engines" is important advisory information that, while not required, is still useful to have.
